### ISSUE TITLE
Can't click on desktop/taskbar etc.

### ISSUE THREAD

**Issue by fersnake** [Ryochan7/DS4Windows#438]
Ok i kinda have this issue, well i had it a couple times and i didnt know what it was and i just restarded my pc, but days ago i build a pc for a friend an i setup everything including cemu and ds4windows (latest) well the thing is that i figure it out why i couldnt click and it was because i unplugged the controller while it was still running ds4windows and give that issue, so in order to prevent that issue i had to reconnect the controller so i could click again and then press the stop button on ds4windows then unplug the controller. im asking this because i need to  do that everytime i wanna stop playing or it's an issue with ds4windows?. 

windows 10 pro 1803
ds4windows 1.5.8

**Comment by Ryochan7** [comment]
I think I have experienced that before as well; it has been rare though. One big problem with the unplug routine is that there is no reasonable way to reset any active output KB+M controls before disconnecting the controller. I have tried to change that in the past but the couple of changes that I have tried made DS4Windows run slower so those changes were scrapped. There is only a loose association between a input device and the output KB+M controls.

I should give this a try again.
